fontbe/src/font.rs

```diff
@@ -0,0 +1,41 @@
+//! Merge tables into a font
+
+use fontdrasil::orchestration::Work;
+use read_fonts::types::Tag;
+use write_fonts::FontBuilder;
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Bytes, Context, WorkId},
+};
+
+struct FontWork {}
+
+pub fn create_font_work() -> Box<BeWork> {
+    Box::new(FontWork {})
+}
+
+// TODO the tables have the tag, but I'm not sure how to access it
+const TABLES_TO_MERGE: &[(WorkId, Tag)] = &[
+    (WorkId::Cmap, Tag::new(b"cmap")),
```

Comment:
You're missing perhaps the most important tables of all, maxp, which defines _how many glyphs_ are in the font :)